lib/rentlinx/validators/url_validator.rb

```diff
@@ -0,0 +1,10 @@
+module Rentlinx
+  class UrlValidator < BaseValidator
+    private
+
+    def validate
+      return if value_blank? || @value.match(%r{^https?:\/\/.*\.(com|net|org|us)\/.*$})
```

Comment:
Is there a good reason to restrict the domain list to only com, net, org and us?